mslib/mswms/mswms.py

```diff
@@ -106,8 +106,8 @@ def main():
         logging.info("Gallery generation done.")
         sys.exit()
 
-    updater.on_update_available.connect(lambda old, new: logging.info(f"MSS can be updated from {old} to {new}.\nRun"
-                                                                      " the --update argument to update the server."))
+    updater.on_update_available.connect(lambda old, new: logging.info("MSS can be updated from %s to %s.\nRun"
```

Comment:
There are several issues with PEP8 compliance here (see failed pipeline).
Here, you need to either align the second string with the line before or join the two strings in a new line (more options are likely possible).
Use pycodestyle or flake8 to test locally. We use some specific configurations to reduce the nagginess of the tools, depending on how you call them, you might get slightly different results than the pipeline. See setup.cfg.